### PR TITLE
Skip clean_removed test

### PR DESCRIPTION
**Pull request by ruflin**
See https://github.com/elastic/beats/issues/7690

### CODE REVIEW

File: filebeat/tests/system/test_registrar.py [hunk 1]
```diff
@@ -851,6 +851,9 @@ def test_clean_removed(self):
         """
         Checks that files which were removed, the state is removed
         """
+
+        raise SkipTest
```
Comment (jsoriano) [inline filebeat/tests/system/test_registrar.py]: Add a comment mentioning the reason why the test is skipped? or with a link to the issue?

**Comment by ruflin** [comment]
@ph Can do, but will it make any difference?

**Comment by ruflin** [comment]
@ph Changed.

**Comment by ph** [comment]
> @ph Can do, but will it make any difference?

it is just cleaner IMHO :)

**Comment by ph** [comment]
@ruflin I have added **needs_backport** label, I think we want to disable that in all the current branches.

**Comment by ruflin** [comment]
backported to 6.x. Would ignore it for now in 6.5.

File: filebeat/tests/system/test_registrar.py [hunk 2]
```diff
@@ -847,10 +848,12 @@ def test_clean_inactive(self):
         else:
             assert data[0]["offset"] == 2
 
+    @unittest.skip("Skipped as flaky: https://github.com/elastic/beats/issues/7690")
```
Comment (jsoriano) [inline filebeat/tests/system/test_registrar.py]: :+1:

File: filebeat/tests/system/test_registrar.py [hunk 3]
```diff
@@ -847,10 +848,12 @@ def test_clean_inactive(self):
...
     def test_clean_removed(self):
         """
         Checks that files which were removed, the state is removed
         """
+
```
Comment (jsoriano) [inline filebeat/tests/system/test_registrar.py]: Nit. Unneeded space added.

Comment (ruflin) [reply filebeat/tests/system/test_registrar.py]: nit ignored as already merged :-)

**Pull request merged**